.codecov.yml

```diff
@@ -13,6 +13,10 @@ coverage:
         threshold: 0.5
         paths:
           - lib/spack/spack/cmd
+      modules:
+        threshold: 0.5
+        paths:
+          - lib/spack/spack/modules
```

Comment:
This gives clear indication, imho, of the coverage of a spack package (in the python sense, so folder with `__init__.py` to be rough :smile: ). Modules are covered 91% apparently,  and if I click on the summary I'll see immediately where I lack testing.